src/cairoWriter/writers/cairoContractWriter.ts

```diff
@@ -150,3 +219,17 @@ export class CairoContractWriter extends CairoASTNodeWriter {
     ];
   }
 }
+
+function getGroupedImports(imports: CairoImportFunctionDefinition[]): string[] {
+  const processedImports: string[] = [];
+  imports.reduce((functionNames: string[], importNode, index) => {
+    functionNames.push(importNode.name);
+    if (importNode.path !== imports[index + 1]?.path) {
+      // TODO: multiple imports are not avaible in cairo1
+      processedImports.push(`use ${importNode.path}::${functionNames.join(', ')}`);
+      functionNames = [];
+    }
+    return functionNames;
+  }, []);
+  return processedImports;
+}
```

Comment:
It was deleted on #968

Reply:
Will be solved when imports go into cairo-1.0 branch and merge it back to this one